microsoftgraph/microsoft-graph-explorer-v2
# fixes errors when displaying profile component

## Overview

Fixes #129
Fixes #73

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Log in 
* Notice that the response component is not populated by your profile picture (Error in issue #73 )
* Run the `me/photo/$value` request
* Notice also that changing the views from response preview to response headers does not cause the app to crash
